momento-sdk/src/main/java/momento/sdk/SimpleCacheClient.java

```diff
@@ -252,6 +243,16 @@ public CompletableFuture<CacheSetResponse> set(String cacheName, String key, Str
     return scsDataClient.set(cacheName, key, value);
   }
 
+  public CompletableFuture<CacheIncrementResponse> increment(
```

Comment:
Would you mind adding Javadocs to these?

Reply:
Yes, sorry I forgot to add that to the commit